test/unit/signer.js

```diff
@@ -409,22 +395,20 @@ describe("Contract: Signer", async () => {
       );
     });
     it("safe should accept signer's signature", async () => {
-      const {
-        data,
-        to,
-      } = await setup.seedFactory.populateTransaction.deploySeed(
-        BENEFICIARY,
-        ADMIN,
-        [PRIME, WETH],
-        [softCap, hardCap],
-        price,
-        startTime,
-        endTime,
-        [vestingDuration, vestingCliff],
-        isPermissioned,
-        fee,
-        metadata
-      );
+      const { data, to } =
+        await setup.seedFactory.populateTransaction.deploySeed(
+          BENEFICIARY,
+          ADMIN,
+          [PRIME, WETH],
+          [softCap, hardCap],
+          price,
+          startTime,
+          endTime,
+          [vestingDuration, vestingCliff],
+          isPermissioned,
+          fee,
+          metadata
+        );
```

Comment:
As I understand most of the changes were done automatically?

Reply:
After configuring prettier I ran `npm run format` and it formatted all specified .js and .sol files according to the prettierrc.json config (and some default rules?). This command can be run manually, or you can potentially configure your editor so that formatting happens automatically upon saving a file (which is what I did w/ VSCode). The result is a standardized format for our codebase.

Reply:
The formatting rules can be customized to a certain degree I guess. I used some standard configuration.

Reply:
Ok, overall seems good. Prettier is definitely a nice practice we need to use more